test/integration/msk.test.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const { expect } = require('chai');
+const log = require('log').get('serverless:test');
+const fixtures = require('../fixtures');
+const { confirmCloudWatchLogs } = require('../utils/misc');
+const {
+  isDependencyStackAvailable,
+  getDependencyStackOutputMap,
+} = require('../utils/cludformation');
+
+const awsRequest = require('@serverless/test/aws-request');
+const crypto = require('crypto');
+const { deployService, removeService } = require('../utils/integration');
+
+describe('AWS - MSK Integration Test', function() {
+  this.timeout(1000 * 60 * 100); // Involves time-taking deploys
+  let stackName;
+  let servicePath;
+  const stage = 'dev';
+
+  const topicName = `msk-topic-${crypto.randomBytes(8).toString('hex')}`;
+
+  before(async function beforeHook() {
+    const isDepsStackAvailable = await isDependencyStackAvailable();
+    if (!isDepsStackAvailable) {
+      log.notice(
+        'CloudFormation stack with integration test dependencies not found. Skipping test.'
+      );
+      this.skip();
```

Comment:
I think we should throw instead. As there's no point in running such test when no setup is made.

What probably could be good is to put this test into `integration/infra-dependent` folder. to clearly separate it from others. Then also via glob patterns we can easily call just non-infra-dependent in some scenarios